Dockerfile

```diff
@@ -89,14 +62,10 @@ FROM system as install
 ARG BTYP
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    libsdl1.2debian libsdl-mixer1.2 libsdl-image1.2 libgtk2.0-0 && \
+    libsdl2-2.0 libsdl2-mixer-2.0 libsdl2-image-2.0 ffmpeg libgtk2.0-0 && \
```

Comment:
Is the whole ffmpeg package needed or just some of its libraries? If so, I would prefer to name those (even if multiple ones) instead of one big package, to avoid unnecessary bloating the DF size, which needs to be pushed and pull multiple times during the GL-CI build and tests. Even though the net-traffic IO is not limited in GL-CI, the additional time needed for up/down loading is lost from the GL-CI minutes (2000 per month for the free plan).

Reply:
I can install the libraries directly which may cause some version mismatch in the future. The libraries are libavcodec57, libavformat57, and libswscale4 which together with there dependencies are still 70Mb where ffmpeg is 130Mb.

In case this will help the automatic build and test system we can implement this as I do not think the versions will change very often and the docker-container is specific for ctp2.

Reply:
Well it is nearly only half the size and it would correspond 1:1 to the build deps in the builder stage, so I would go with just the libs and fix the package names in case they change.

Reply:
Replaced ffmpeg with the needed packages.